readthedocs/urls.py

```diff
@@ -77,9 +77,10 @@
     # Keep the `doc_search` at root level, so the test does not fail for other API
     url(r'^api/v2/docsearch/$', PageSearchAPIView.as_view(), name='doc_search'),
     url(
-        r'^api-auth/',
+        r'^api/auth/',
```

Comment:
Is this backwards compat?

Reply:
No, it's not.

I think I left this change by mistake since it's not really necessary but I was testing another thing and I wanted to keep the pattern.

Although, I think we don't care since this is not really used as far as I know.